lib/tasks/usps_enrollment.rake

```diff
@@ -0,0 +1,71 @@
+namespace :usps_enrollment do
+
+  desc 'Pass an enrollment'
+  task pass: :environment do
+    # FIXME: Check if we're in prod, and yell at the user + fail if so!
+
+    enrollment = InPersonEnrollment.find_by_enrollment_code(ENV.fetch('ENROLLMENT_CODE', ''))
+    raise "No such enrollment" unless enrollment.present?
+
+    # Do we still need this?
+    enrollment.update(unique_id: enrollment.usps_unique_id) if enrollment.unique_id.blank?
+
+    mock_response = {
+      # TODO: This should actually be in CST apparently.
+      'transactionEndDateTime' => Time.zone.now.strftime('%m/%d/%Y %H%M%S'),
+      'fromRakeTask' => 'yes'
+    }
+
+    # This is probably the most illegal thing I've ever done.
+    job = GetUspsProofingResultsJob.new
+    job.instance_variable_set('@enrollment_outcomes', { enrollments_passed: 0 })
+    job.send(
+      :handle_successful_status_update,
+      enrollment,
+      mock_response
+    )
```

Comment:
So, my thinking here is that I want us to take the same codepath as if we hit their API and got a result. This task emulates getting into `check_enrollment` and then into `process_enrollment_response`, which calls `handle_successful_status_update`.

Those are pretty reasonably private methods, so we go for a dirty approach.

![so dirty](https://64.media.tumblr.com/3e5de56279d7aa409c09ab8601974ebe/b0ea9792e807e275-a1/s540x810/8e3f29a4c6ce49ecceb47c2fb796a2550c8952ad.gifv)